specs/sharding/beacon-chain.md

```diff
@@ -448,15 +469,19 @@ def get_start_shard(state: BeaconState, slot: Slot) -> Shard:
 ```python
 def compute_shard_from_committee_index(state: BeaconState, slot: Slot, index: CommitteeIndex) -> Shard:
     active_shards = get_active_shard_count(state, compute_epoch_at_slot(slot))
+    assert index < active_shards
```

Comment:
Formally `index` and `active_shards` are incompatible types and comparison is not formally correct. 
If the `index` passed is valid in the context of `(state, slot)` then this assertions should never fail.

Reply:
Both are derived from a `uint64`, I think the comparison is clear enough. Attestations (and thus committee index) have been verified before the shard processing, but we may want to use this function elsewhere, so I think we still need the assertion.